tiatoolbox/wsicore/wsireader.py

```diff
@@ -486,7 +504,7 @@ def find_read_rect_params(
         location: IntPair,
         size: IntPair,
         resolution: Resolution,
-        units: str,
+        units: Units,
```

Comment:
These are minor edits which can be done in a separate PR.

Reply:
Moved to https://github.com/TissueImageAnalytics/tiatoolbox/pull/598